commit 573eeddd45386418c70adc5d501353e38a53bd75
Author: Lyude Paul <lyude@redhat.com>
Date:   Wed Aug 15 15:00:12 2018 -0400

    drm/nouveau: Remove duplicate poll_enable() in pmops_runtime_suspend()
    
    commit 611ce855420a6e8b9ff47af5f47431d52c7709f8 upstream.
    
    Since actual hotplug notifications don't get disabled until
    nouveau_display_fini() is called, all this will do is cause any hotplugs
    that happen between this drm_kms_helper_poll_disable() call and the
    actual hotplug disablement to potentially be dropped if ACPI isn't
    around to help us.
    
    Signed-off-by: Lyude Paul <lyude@redhat.com>
    Acked-by: Karol Herbst <kherbst@redhat.com>
    Acked-by: Daniel Vetter <daniel@ffwll.ch>
    Cc: stable@vger.kernel.org
    Cc: Lukas Wunner <lukas@wunner.de>
    Signed-off-by: Ben Skeggs <bskeggs@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/nouveau/nouveau_drm.c b/drivers/gpu/drm/nouveau/nouveau_drm.c
index c7ec86d6c3c9..5fdc1fbe2ee5 100644
--- a/drivers/gpu/drm/nouveau/nouveau_drm.c
+++ b/drivers/gpu/drm/nouveau/nouveau_drm.c
@@ -835,7 +835,6 @@ nouveau_pmops_runtime_suspend(struct device *dev)
 		return -EBUSY;
 	}
 
-	drm_kms_helper_poll_disable(drm_dev);
 	nouveau_switcheroo_optimus_dsm();
 	ret = nouveau_do_suspend(drm_dev, true);
 	pci_save_state(pdev);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1063_573eeddd45386418c70adc5d501353e38a53bd75.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1063_573eeddd45386418c70adc5d501353e38a53bd75.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
